Exygy/askdarcel-web
# Content template with styles

# Pull Request Template

## Issue Overview

This PR addresses #issue


## Description

[Ticket ](https://www.notion.so/exygy/fde60299a9f34cc8a3f81fa81c8395a3?v=4b5977a90928486085512c06e2d1db2a&p=d13f1247095e452b8d15f50667c82f95&pm=s)


## How Can This Be Tested/Reviewed?

- Render app, go to About page - check that it looks similar enough to figma
- Review code
- Note: Need to sync with design on some of the tablet break points - not sure if we need to add constraints to the content length or the front end, or both. I predict we'll need to clean up a lot of the tablet breakpoints for the app. I'm down to merge in features and tackle that in a separate task, to keep momentum going.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
